### DISCUSSION TITLE
reset when using INA219

### DISCUSSION THREAD

**Discussion by KarlEffinger** [RolandRust/HaniMandlWroom#9] [Q&A]
Hi Roland, hi everyone,
first of all: I am deeply impressed by this work. I built a "standard" hanimandl and it works perfectly. But the large display, the smooth menu handling, the enhanced information screens - just great!
I set up a pcb for my own (design) needs and built it up, but now I am having some trouble: whenever I have the INA219 board in place and press the "start" button, the servo moves and the esp reboots. When bridging the Vin and Vout and not connecting the other INA pins, everything works like a charm.
Any idea what the reason for this could be? I was trying to set up the debug level to 5 in order to get some more information on where exactly the reset occurs, but then I run into compiling errors (I just placed the corresponding Issue)...
Any help appreciated!
Karl

**Comment by RolandRust** [comment]
Hi Karl,
If I understood right, then the Servo is moving without crash if you bridge the VIN and VOUT from the INA (J11.5 and J11.6). 
What is happening if you do not press the Start Button? Does you see some current in the display or is it 0mA? And was are your settings in the INA menu? Usually I have only a crash from the ESP if I use a to week Power Supply (the TFT use much more power then the OLED). But If the ESP not crash without INA and the Servo is moving, then it is not a problem from the power supply.
Does it also crash if the INA is set off in the menu?  If the INA is setting of in the menu, then the SW should only communicate one with the INA in the startup and then the HM should behave like the IN and OUT are bridged with the only different that R7 (0.1 Ohm) is in the track.
What can be the problem:
1. SW bug (I hope not :-)) 
2. R7 is more then 0.1 Ohm
3. Short between SDA and SDL (but then you shoould not see the INA menu)

Can you try to set the INA in the menu off and look if the ESP crash also? 

BR Roli

P.S. I think the INA has no debug function implemented if they have some, then it in the Level 1. This Levels are from the original SW and I never used an other level then 1.

**Comment by KarlEffinger** [comment]
Hi Roli,
the "if I use a too week Power Supply" was the key. When I attached a 3A USB Power Supply, everything works just well. The reboots only occur, when connected to my laptop (which I always had so far, for debugging purposes...).
So - fine for me! And thanks for your promt support!!

Grüße
Karl
